examples/rich/rich.html

```diff
@@ -89,6 +89,17 @@
             }
           }
 
+          var customBlockMap = Object.assign({}, DefaultDraftBlock, {
```

Comment:
There is an issue open for this: https://github.com/facebook/draft-js/issues/52. @sstur is going to take a crack at it.